frontend/src/templates/table_top_statements.html

```diff
@@ -0,0 +1,11 @@
+<table class="frontTable noBorder">
+    <thead><th>Name</th><th>Calls</th><th>Total Time</th><th>Avg. Time</th></thead>
+    {% for row in data: %}
+    <tr>
+        <td><a href="/perfstatstatements/graph/{{hostuiname}}/{{row.query_id}}">{{ row.query|e }}</a></td>
+        <td align="right">{{ row.calls }}</td>
```

Comment:
the ```align``` attribute is long deprecated, I think: http://www.w3.org/TR/html401/present/graphics.html#h-15.1.2